hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -826,12 +917,136 @@ def _get_charge_limits(self, token: Token, vehicle: Vehicle) -> dict:
             url, headers=self._get_authenticated_headers(token)
         ).json()
         _LOGGER.debug(f"{DOMAIN} - Get Charging Limits Response: {response}")
-        _check_response_for_errors()
+        _check_response_for_errors(response)
         # API sometimes returns multiple entries per plug type and they conflict.
         # The car itself says the last entry per plug type is the truth when tested (EU Ioniq Electric Facelift MY 2019)
         if response["resMsg"] is not None:
             return response["resMsg"]
 
+    def _get_trip_info(
+        self,
+        token: Token,
+        vehicle: Vehicle,
+        date_string: str,
+        trip_period_type: int,
+    ) -> dict:
+        url = self.SPA_API_URL + "vehicles/" + vehicle.id + "/tripinfo"
+        if trip_period_type == 0:  # month
+            if len(date_string) != 6:
+                raise APIError(
```

Comment:
I think a `ValueError` would be more appropriate, since it is an input. I'd like to keep APIError for things returned by the API.

Reply:
As far as I can see ValueError is nowhere used and is not declared, I can also remove the check and let the API return the error? Or do you want me to introduce ValueError?

Reply:
ValueError is a built-in Python exception, you don't need to create/import it.
And I think it is good to perform value checks before performing API calls. The less unnecessary/invalid calls, the better. But it's up to you.

Reply:
I think I leave it without the checks, because nowhere else in the library this is done.